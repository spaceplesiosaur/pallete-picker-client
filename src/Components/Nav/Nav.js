import React from 'react';
import './Nav.scss';

const Nav = () => {
    return (
        <nav>
           <h1>Palette Picker</h1>
        </nav>
    );
};

export default Nav;