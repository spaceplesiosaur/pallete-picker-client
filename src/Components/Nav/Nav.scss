nav {
    background-color: #B3B3F1;
    margin: 0;
    padding: 5px;
    h1 {
        font-family: 'Zhi Mang Xing', cursive;
        padding: 10px;
        font-size: 4em;
        text-align: center;
        color: #dc65c6;
        text-shadow: 2px 2px #607D8B;
    }
}