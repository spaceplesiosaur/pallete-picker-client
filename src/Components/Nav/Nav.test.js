import React from 'react';
import { shallow } from 'enzyme';
import Nav from './Nav'

describe('Nav', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Nav />)
  })

  it('should render with the correct properites', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
