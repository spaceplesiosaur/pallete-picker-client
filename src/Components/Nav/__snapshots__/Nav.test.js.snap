// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render with the correct properites 1`] = `
<nav>
  <h1>
    Palette Picker
  </h1>
</nav>
`;
